third-party-programs/Velocity-Bench/cudaSift/README.md

```diff
@@ -100,6 +101,65 @@ Now you can see the migrated files in the `out` folder as follow:
       └── matching.dp.cpp
 
    ```
+
+## 3.2 Migrate the source code and CMake build script
+In section 3.2, after running the migration command:
```

Comment:
don't ref to 3.1,  just list the code migration command again.

Reply:
okay